File: LLAndroidJavaSamples/app/src/main/java/com/myco/MyApplication.java
package com.myco;

import android.app.Application;
import android.util.Log;

import com.locuslabs.sdk.llpublic.LLConfiguration;
import com.locuslabs.sdk.llpublic.LLMapPackFinder;
import com.locuslabs.sdk.llpublic.LLOnUnpackCallback;

import java.util.ArrayList;
import java.util.List;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();

        LLConfiguration.Companion.getSingleton().setApplicationContext(getApplicationContext());
        LLConfiguration.Companion.getSingleton().setAccountID("A11F4Y6SZRXH4X");
    }
}

File: LLAndroidKotlinSamples/app/src/main/java/com/myco/MyApplication.kt
package com.myco

import android.app.Application
import com.locuslabs.sdk.llpublic.LLConfiguration

class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        LLConfiguration.singleton.applicationContext = applicationContext
        LLConfiguration.singleton.accountID = "A11F4Y6SZRXH4X"
    }
}